src/components/chat/ChatRoom.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import { createStructuredSelector } from 'reselect';
+import { connect } from 'react-redux';
+
+import checkAuth from '../requireAuth';
+import {
+  launchPushMessages,
+  requestMessages,
+} from '../../actions/messagesActions';
+import MessageForm from './MessageForm';
+import Message from './Message';
+import { getUser } from '../../reducers/userReducer';
+import { getLoading } from '../../reducers/loadingReducer';
+import { getMessages } from '../../reducers/messagesReducer';
+import { isLogged } from '../../reducers/authReducer';
+
+class ChatRoom extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      message: { value: '' },
+    };
+  }
+
+  componentWillMount() {
+    const { requestMessages, isLogged } = this.props;
+    if(isLogged) {
+      requestMessages();
+    }
+  }
+
+  componentWillReceiveProps({ loading }) {
+    const { loading: previousLoading } = this.props;
+    if (previousLoading !== loading && !loading) {
+      this.setState({ message: { value: '' } });
+    }
+  }
+
+  updateMessage = e => {
+    let message = this.state.message;
+    message[e.target.name] = e.target.value;
```

Comment:
⚠️ I think you are implicitly mutating the state here.

Indeed:
```js
const message = this.state.message;
message['foo'] = 'bar';
this.setState({message: message})
this.state.message === message //=> true
```

https://medium.com/pro-react/a-brief-talk-about-immutability-and-react-s-helpers-70919ab8ae7c